src/environment.cpp

```diff
@@ -426,13 +426,15 @@ bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, v3s16
 	return true;
 }
 
-void ServerEnvironment::kickAllPlayers(const std::string &reason)
+void ServerEnvironment::kickAllPlayers(u8 reason,
```

Comment:
I don't like including headers into other headers just for a single type, because its added dependencies, so rebuild time can increase in future somebody changes something in `networkprotocol.h`, also people can just look the value up :), but if you both want it, I will do it.